cmd/ebrelayer/relayer/cosmosReplay.go

```diff
@@ -276,7 +276,43 @@ func (sub CosmosSub) getAllSignSigature(accAddress sdk.AccAddress, fromBlock int
 		}
 	}
 
-	return claimArray
+	return claims
+}
+
+// GetLastNonceSubmitted get last nonce submitted in cosmos bridge contract
+func (sub CosmosSub) GetLastNonceSubmitted() (*big.Int, error) {
+	client, _, target, err := tryInitRelayConfig(sub)
+	if err != nil {
+		sub.SugaredLogger.Errorw("failed in init relay config.",
+			errorMessageKey, err.Error())
+		return nil, err
+	}
+
+	// Initialize CosmosBridge instance
+	cosmosBridgeInstance, err := cosmosbridge.NewCosmosBridge(target, client)
+	if err != nil {
+		sub.SugaredLogger.Errorw("failed to get cosmosBridge instance.",
+			errorMessageKey, err.Error())
+		return nil, err
+	}
+	return cosmosBridgeInstance.LastNonceSubmitted(nil)
```

Comment:
The variable has a lowercase L to start. Have you verified this returns the proper number from the smart contract?

Reply:
but I can't access to the varialbe directly, must use the function with capital. didn't do any integration test yet.